lib/hanami/action/cache/conditional_get.rb

```diff
@@ -41,7 +41,7 @@ def fresh?
         # @since 0.3.0
         # @api private
         def header
-          { ETAG => @value } if none_match
```

Comment:
Please, can you add `if @value`? if we send a ETAG value without value there will be a rack lint error because ETAG will be nil. IMO we should send this ETAG if we are using `fresh` method with `:etag` option.